packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Prepare your SCIM API service
+excerpt: Prepare a SCIM-compliant API server to host your SCIM service, and test it to make sure it is working correctly.
+meta:
+  - name: description
+    content: Prepare a SCIM-compliant API server to host your SCIM service, and test it to make sure it is working correctly.
+layout: Guides
+---
+
+This guide teaches you how to prepare a SCIM-compliant API server to host your SCIM service, and test it to make sure it is working correctly.
+
+## Preparation
+
+The first step in delivering your SCIM integration is preparing a SCIM-compliant API server to host your SCIM service.
+
+Okta supports lifecycle provisioning using either Version 2.0 and Version 1.1 of the SCIM protocol.
+
+If your service already supports the SCIM protocol, it is important that you review the Okta SCIM reference documentation to understand the specifics of how Okta implements the SCIM protocol:
+
+- [SCIM Reference](/docs/reference/scim/)
+
+If you haven't yet implemented SCIM, Okta recommends that you use Version 2.0 of the SCIM protocol.
+
+## Features
+
+Another important part of the planning process is determining which Okta provisioning features your SCIM API can or should support. At a minimum, Okta requires that your SCIM Version 2.0 API implements the following features.
```

Comment:
Do these "minimum requirements" merit inclusion under **What you need** in the nut facts?

Reply:
This is a good point. I'm going to add a link to this place in the overview nut facts.